src/components/structures/MessagePanel.js

```diff
@@ -962,6 +962,17 @@ class MemberGrouper {
     }
 
     add(ev) {
+        if (ev.getType() === 'm.room.member') {
+            // We'll just double check that it's worth our time to do so...
+            if (ev.getPrevContent()) {
+                const membershipChange = ev.getPrevContent()['membership'] !== ev.getContent()['membership'];
+                const displayNameChange = ev.getPrevContent()['displayname'] !== ev.getContent()['displayname'];
+                const avatarChange = ev.getPrevContent()['avatar_url'] !== ev.getContent()['avatar_url'];
+                if (!membershipChange && !displayNameChange && !avatarChange) {
```

Comment:
it should be shown when show hidden events is enabled though, no? This should perhaps re-use one of the multitude of hidden-event check methods we have

Reply:
Not sure that's a feature or a bug tbh. Where are these checks?

Reply:
https://github.com/matrix-org/matrix-react-sdk/blob/f84795ebaeefe3cf44d858ca9c2ffb25094371c3/src/TextForEvent.js#L80 claims feature